build/yamls/antrea-windows-containerd.yml

```diff
@@ -21,6 +21,39 @@ data:
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
     & "$mountPath/k/antrea/bin/antrea-agent.exe" --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
+
```

Comment:
We should add a flag that allows user to decide whether to enable OVS running in a container. One of the reasons is we are uncertain if all previous pods can smoothly upgrade to this version.

Reply:
thanks, added a new yaml file.